### PR TITLE
Add a proper sign out test

### PR DESCRIPTION
**Pull request by NeilvB**
Previously we were manipulating the cookie within the test setup rather than properly testing the application sign out logic.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
...
+    "test:unit": "NEXT_PUBLIC_ENV_NAME=test node --expose-gc ./node_modules/.bin/jest --runInBand --logHeapUsage",
     "lint": "eslint . --max-warnings 0 --ext .js,.jsx && echo 'Lint complete.'",
-    "test:e2e": "PORT=5001 start-test 5001 cy:run",
-    "e2e:audit": "PORT=5001 start-test 5001 cy:run:audit",
-    "e2e:interactive": "PORT=5001 start-test 5001 cy:open",
-    "e2e:server": "PORT=5001 start-test 5001",
+    "test:e2e": "NEXT_PUBLIC_ENV_NAME=test PORT=5001 start-test 5001 cy:run",
+    "e2e:audit": "NEXT_PUBLIC_ENV_NAME=test PORT=5001 start-test 5001 cy:run:audit",
+    "e2e:interactive": "NEXT_PUBLIC_ENV_NAME=test PORT=5001 start-test 5001 cy:open",
+    "e2e:server": "NEXT_PUBLIC_ENV_NAME=test PORT=5001 start-test 5001",
```
Comment (NeilvB) [inline package.json:20]: These will all be derived from `NODE_ENV` once we switch to dotenv-flow. There is an open PR for that but it will need updating.

Comment (Asura5) [reply package.json:20]: Do you need && before jest to execute the shell param store or is that not needed?

Comment (NeilvB) [reply package.json:20]: I don't think it's needed - just checked and code executed from Jest can access this value.

**Review by Asura5 — APPROVED** [review]

**Review by teodor-hristoforov-hackney — APPROVED** [review]

**Comment by NeilvB** [comment]
Thanks for the help with reviews on this.

**Pull request merged**